src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -21,30 +21,30 @@
     private double actualResult;
 
     @BeforeAll
-    static void setUpCalculator() {
+    static void setUp_Calculator() {
         calculator = new Calculator();
     }
 
     @Test
-    public void additionWithTwoPositiveOperands_Ok() {
+    public void addition_TwoPositiveOperands_isOk() {
```

Comment:
```suggestion
    public void calculate_TwoPositiveOperands_isOk() {
```
we need to specify what METHOD we test